### DISCUSSION TITLE
move `eralchemy` project to this organization ?

### DISCUSSION THREAD

**Discussion by 12rambau** [sqlalchemy/sqlalchemy#11553] [Ideas]
Hi `sqlalchemy` maintainers and thanks for your dedicated work! 

I recently started to use your tool and let's face it, it's very, very good. One functionality that I was missing was the capacity to draw relationship graphs from `Base` and I was scouting the jungle of existing project to find this little gem: [`eralchemy`](https://github.com/Alexis-benoist/eralchemy). The project, owned by @Alexis-benoist has been on stale for several years but in fact a dedicated maintainer had restarted it in [`eralchemy2`](https://github.com/maurerle/eralchemy2). 

In the cruel world of OSS development, maintainers time is the main resources and when the owner gets overloaded, packages are forgotten and left for dead leading to complicated to follow version number included in the name like pyPDF, pyPDF4, etc.... 

As I really like this project I contacted the original owner to get maintainer rights on both the pypi and github projects. @maurerle and I will update it to make it shine as a modern python lib but it will remain at risk. What I did for other projects is handing it over the most related big organization to give it more visibility for helpers. In this case the big org is .... you. 

Would you accept that we hand over the project to you and continue maintaining from inside your organization ? 
If it can help our case, we are not talking about a little side project, it cumulated more than 1k stars and around 15k downloads/month so I think it would not tarnish `sqlalchemy` and `alembic` if sitted next to them.

**Comment by zzzeek** [comment]
hiya -

myself and @CaselIT are the maintainers of every project under the "sqlalchemy" organization and all of them are basically my own projects.    I would not have the bandwidth to have other projects start building under the same org, and if you folks as maintainers of eralchemy are concerned about keeping maintenance of the project going, I wish you the best but I personally can't take on the role of keeping that maintenance going myself.    There's a lot of important SQLAlchemy-related projects like geoalchemy2, sqlmodel, sqlalchemy-utils, alembic-utils, and it's just a matter of course that the maintenance / popularity of these projects will come and go based on trends; they're not "mine" so I currently dont have a place for any of them within the SQLAlchemy org directly right now.

**Reply by 12rambau** [reply]
ok I thought it was more like a [`sphinx-contrib`](https://github.com/sphinx-contrib), [`copier-org`](https://github.com/copier-org) or [`jupyter`](https://github.com/jupyter) org that aggregate related project and people from a flagship project. I was of course offering to participate to the organization as well as long as possible (this is what I did in the quoted orgs) to maintain the said project. I'm not unloaded the responsability to you, but more making the project anonymous.  

If it remains a clear no, feel free to close this thread and we'll do our best from our side.

**Reply by zzzeek** [reply]
I would say that, for now, it's more analogous to [pallets](https://palletsprojects.com/) - a collection of projects that originated with the same developer.

this is only because as mentioned, me and @CaselIT are mostly it for now and sqlalchemy org itself / infra I do most of that personally.

I understand you're looking to maintain eralchemy "as long as possible" but that works just as well with eralchemy not being in the SQLAlchemy project.  it's the after "as long as possible" part that I personally can't commit to, that's all.  noting, I've been in this community since 2005.  I have seen lots of very big, very major projects go dormant, seen some of them come back, others not.   nothing is forever, but having a large legacy/dormant project on your hands is a major inconvenience.

**Reply by 12rambau** [reply]
Very clear, thanks for responding so fast!
